training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml

```diff
@@ -441,6 +440,32 @@ inputs:
     optional: true
     description: Additional parameters as JSON serielized string
 
+  # Compute parameters
+  compute_model_import:
+    type: string
+    optional: true
+    default: serverless
+    description: compute to be used for model_import eg. provide 'FT-Cluster' if
```

Comment:
In the description, can you mention that if compute cluster is mentioned here, the instance type will be ignored?